inst/www/shared/shiny_scss/shiny.scss

```diff
@@ -0,0 +1,436 @@
+/*************************************************
```

Comment:
Now that we have this file, we can get rid of shiny.css, right?

Reply:
We can if you want, but I was thinking it'd be better practice to have a build script to compile this to css that we can serve up statically if no `bs_theme()` is used, but I haven't gotten to that yet.